packages/functional-tests/tests/react-conversion/signinTotp.spec.ts

```diff
@@ -6,47 +6,78 @@ import { test, expect } from '../../lib/fixtures/standard';
 
 test.describe('severity-1 #smoke', () => {
   test.describe('two step auth', () => {
-    test.beforeEach(async () => {
+    test.beforeEach(async ({ pages: { configPage } }) => {
+      const config = await configPage.getConfig();
+      test.skip(config.showReactApp.signInRoutes !== true);
```

Comment:
This might need to temporarily be `inlineTotpRoutes` though we have a ticket to roll this totp setup route into the `signInRoutes` before we flip the switch for staging: https://mozilla-hub.atlassian.net/browse/FXA-9325

Reply:
Actually I looked too quickly. Looks like this test goes through the totp setup via settings, then using the totp via signin. `signInRoutes` is correct here.
Maybe the setup should be in a fixture vs. in the test steps since it's not affected by the conversion? This test file should probably only be testing _using_ the totp.

Reply:
I need a bit more information. Are you indicating that we could use an alternative set of steps in this test? Or that there is a common setup here that could be better encapsulated and re-used? or both?

I have found an alternative to using the react flag as a fix, I have a preference for it as it accommodates all environments; however, it will loose it's usefulness as the React migration continues (https://github.com/mozilla/fxa/pull/16621/commits/ee7e9ee6da8cd4a9beff23a806a4817f9cdb7a76).  Let me know what you think.

Reply:
Apologies, I bundled a few separate thoughts in one comment.

- My initial comment was based on a misunderstanding - I had read too quickly and thought the test was using the newly converted "inline_totp_setup" to enable 2FA on the account. 
- If the test had used the inline totp setup steps, it would have required a different flag check ("inlineTotpRoutes").
- The only react route used in this test is the signInRoutes => the config check is correct
- In this case, 2FA is enabled via /settings, which is not impacted by the react conversion.
- Since settings is not part of the react conversion experiment (it has been stable in react for a few years), I wondered if enabling 2FA on a test account could be part of a fixture.
- This would reduce the test length and focus specifically on checking if sign in with TOTP is working as expected.
- This fixture could be shared by any backbone or react tests that require an account with 2FA enabled.

If we remove the flag check entirely, that means this test would run for both backbone and react. Is that preferred, or should it only run for react since it's in the react-conversion folder? If we keep the config check vs using an `or` on the locator, cleanup might also be easier once we sunset the backbone-specific tests.